src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.tsx

```diff
@@ -60,24 +62,59 @@ const ListedBlastSubmission = (props: Props) => {
     };
   });
 
-  const sequenceBoxes = jobsGroupedBySequence.map(({ sequence, jobs }) => (
-    <SequenceBox key={sequence.id} sequence={sequence} jobs={jobs} />
-  ));
+  let sequenceContent = null;
+  if (isExpanded || sequences.length === 1) {
+    sequenceContent = jobsGroupedBySequence.map(({ sequence, jobs }) => (
+      <SequenceBox key={sequence.id} sequence={sequence} jobs={jobs} />
+    ));
+  } else {
+    sequenceContent = <CollapsedSequencesBox submission={submission} />;
+  }
 
   return (
     <div className={styles.listedBlastSubmission}>
-      <Header {...props} isAnyJobRunning={isAnyJobRunning} />
-      {sequenceBoxes}
+      <Header
+        {...props}
+        isAnyJobRunning={isAnyJobRunning}
+        setExpanded={setExpanded}
+        isExpanded={isExpanded}
+        totalSequences={sequences.length}
+      />
+      {sequenceContent}
+    </div>
+  );
+};
+
+const CollapsedSequencesBox = (props: Props) => {
+  const { submission } = props;
+  const sequences = submission.submittedData.sequences;
+  const allJobs = submission.results;
+
+  const totalSequences = sequences.length;
+  const totalSpecies = allJobs.length / totalSequences;
```

Comment:
Interesting. But don't we have access to the species themselves on the submission? Why not look up how many there are? Would be more direct.

Reply:
That's done. Lack of knowledge!